### ISSUE TITLE
Implement `exhibits_url` to fetch exhibits from an external URL

### ISSUE THREAD

**Issue by krassowski** [nebari-dev/jupyterlab-gallery#15]
To make the configuration easier, `GalleryManager.exhibits_url` should be available allowing to fetch the exhibits from a JSON or yaml file

**Comment by krassowski** [comment]
Ability to change examples without redeploying would be really useful for conferences where multiple instructors contribute example repositories asynchronously and the person compiling the list may not necessarily have rights to redeploy.

**Comment by andrewfulton9** [comment]
I have a couple of questions about this:

Is the expectation here that the a get request to the URL would directly return a json object? 

Should `exhibits_url` override exhibits defined in `GalleryManager.exhibits`, or maybe append to them?

**Comment by krassowski** [comment]
> Is the expectation here that the a get request to the URL would directly return a json object?

Yes. In practice this could be a file in a GitHub repo but linked using raw URL pointing to a branch.

> Should exhibits_url override exhibits defined in GalleryManager.exhibits, or maybe append to them?

No strong opinion. What do you think would be better?

**Comment by andrewfulton9** [comment]
> No strong opinion. What do you think would be better?

I don't think I have a string opinion on which would be better. I am leaning towards appending the exhibits from `exhibits_url` though. Is it possible to make traitlet behavior dependent on other traitlets? If we were going to have `exhibits_url` override `exhibits`, then I would probably want to make it so both can not be set in the configuration in the first place. On the other hand, I could probably add a setting that determines which is displayed. I think the simplest and the best UX would just to combine the exhibits though.
